<launch>
    <!-- 二维码降落 -->
	<!-- 
	search_alt_：预定搜索高度
	markers_id_ ：ar_markers 二维码数字
    desire_pose_x、desire_pose_y、desire_pose_z：期望的飞机相对降落板的位置
	PidXY_p：x,y方向p项系数
	PidXY_d：x,y方向d项系数
	PidXY_i：x,y方向i项系数
	PidZ_p：z方向p项系数
	PidZ_d：z方向d项系数
	PidZ_i：z方向i项系数
	PidYaw_p：yaw方向的p系数
	PidYaw_d：yaw方向的d系数
	PidYaw_i：yaw方向的i系数

	关于其中的yaw角度，yaw为0度的情况是：摄像头图像左上为"0"markers时
    desire_yaw_:弧度值
    -->
	<node pkg="px4_control" type="landing_quadrotor_node" name="landing_quadrotor_node" output="screen">
		<param name="search_alt_" value = "4"/>	
		<param name="markers_id_" value = "96"/>

		<param name="desire_pose_x" value = "0"/>	
		<param name="desire_pose_y" value = "0"/>	
		<param name="desire_pose_z" value = "0.2"/>
		<param name="desire_yaw_" value = "0"/>

		<param name="PidXY_p" value = "1"/>	
		<param name="PidXY_d" value = "0.1"/>	
		<param name="PidXY_i" value = "0.02"/>	
		<param name="PidZ_p" value = "0.2"/>	
		<param name="PidZ_d" value = "0.1"/>	
		<param name="PidZ_i" value = "0"/>	
		<param name="PidYaw_p" value = "0.1"/>	
		<param name="PidYaw_d" value = "0"/>	
		<param name="PidYaw_i" value = "0"/>		
	</node>

    <!-- 启动二维码识别节点 -->
    <!-- <include file="$(find ar_track_alvar)/launch/ar_track_sim_camera.launch" >
    </include> -->
    <include file="$(find apriltag_ros)/launch/apriltag_sim_camera.launch" >
    </include>

    <!-- rviz view /-->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find robot_vision)/rviz/apriltag_sim_camera.rviz"/>
</launch>
